.github/workflows/prefect-client.yaml

```
@@ -28,6 +28,10 @@ on:
         description: "The name for the build prefect-client artifact"
         default: "prefect-client-pypi-dists"
         type: string
+      python-versions:
+        description: "What versions of Python to run against"
+        default: ""
```

Does this default work?
```suggestion
        default: '["3.8", "3.9", "3.10", "3.11", "3.12"]'
```
If it does, that would simplify the `python-version` determination on line 43.

---

It doesn't unfortunately because `inputs` are only populated when the workflow is triggered via `workflow_call`; otherwise the `inputs` are null'ed.  This default is only set on a per-call basis.

---

I guess I could just make this required on workflow_call; it's only used by the one other workflow but might as well for clarity.

---

updated